train_segmentation.py

```diff
@@ -325,6 +325,12 @@ def training(train_loader,
                                ep_num=ep_idx + 1,
                                scale=scale)
 
+        # FIXME: binary vs multiclass loss don't expect same dimensions
+        try:
+            if criterion.mode == 'binary':
+                outputs = outputs[:, -1, ...]
```

Comment:
@victorlazio109 did you run into this "dimensionality" problem when using binary losses? When testing out, I noticed the binary losses only accept logits without the "backgound". Therefore, the prediction array must be indexed as shown in this bit of ugly code from [1,2,256,256] (each number being [batch size, num_classes, width, height]) to [1, 256, 256], discarding the background array and therefore deflating the dimensions to match those of the labels [1, 256, 256]

Reply:
resolved with my last PR, confirm if you can